### PR TITLE
Rename (qubits+cbits) to labels

### PR DESCRIPTION
**Pull request by daxfohl**
Follow-up to #4627 that renames diagram lines from "qubits" to "labels", since they now contain control keys as well.

### CODE REVIEW

**Comment by daxfohl** [comment]
@95-martin-orion ping this should be quick

File: cirq-core/cirq/protocols/circuit_diagram_info_protocol.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ class CircuitDiagramInfoArgs:
             properly handle unicode characters.
         precision: The number of digits after the decimal to show for numbers in
             the text diagram. None means use full precision.
-        qubit_map: The map from qubits to diagram positions.
+        label_map: The map from label entities to diagram positions.
```
Comment (95-martin-orion) [inline cirq-core/cirq/protocols/circuit_diagram_info_protocol.py:191]: I think it's worth defining "label entity" here (IIUC, it's a `Qid` or a `MeasurementKey`).

File: cirq-core/cirq/circuits/circuit.py [hunk 2]
```diff
@@ -1153,32 +1153,32 @@ def to_text_diagram_drawer(
         qubits: Tuple[Any, ...] = ops.QubitOrder.as_qubit_order(qubit_order).order_for(
             self.all_qubits()
         )
-        cbits = tuple(
+        cbits: Tuple[Any, ...] = tuple(
```
Comment (95-martin-orion) [inline cirq-core/cirq/circuits/circuit.py:1156]: Is it possible for `qubits` and `cbits` to have types `Tuple[Qid, ...]` and `Tuple[MeasurementKey, ...]`, respectively? Similarly, can `labels` have type `Tuple[Union[Qid, MeasurementKey], ...]`?

**Review by 95-martin-orion — CHANGES_REQUESTED** [review]
Doesn't look like there are any public API changes. Couple of type-annotation requests, but otherwise this is ready.

**Review by 95-martin-orion — APPROVED** [review]

**Pull request merged**